packages/jaeger-ui/src/components/Monitor/ServicesView/serviceGraph.tsx

```diff
@@ -163,7 +163,7 @@ export class ServiceGraphImpl extends React.PureComponent<TProps> {
         <YAxis />
         {this.renderLines()}
         <Crosshair values={this.state.crosshairValues}>
-          <div style={{ width: 140 }}>
+          <div style={{ width: 140, color: '#6B6B76' }}>
```

Comment:
Shouldn't this be changed in a style sheet somewhere instead of local override?

Reply:
moved the width and the color to css.
thanks